### PR TITLE
Refactor/balance presenter

### PR DESCRIPTION
**Pull request by ditn**
Next up: Tests

### CODE REVIEW

File: app/src/main/java/piuk/blockchain/android/data/datamanagers/PromptManager.kt [hunk 1]
```diff
@@ -58,11 +58,11 @@ class PromptManager constructor(private val prefsUtil: PrefsUtil
     }
 
     private fun isSurveyComplete(): Boolean {
-        return prefsUtil.getValue(PrefsUtil.KEY_SURVEY_COMPLETED, false);
```
Comment (riaanjvos) [inline app/src/main/java/piuk/blockchain/android/data/datamanagers/PromptManager.kt]: ; old habits die hard

File: app/src/main/java/piuk/blockchain/android/ui/account/AccountEditViewModel.java [hunk 2]
```diff
@@ -165,6 +160,7 @@ public void onViewReady() {
 
         } else if (addressIndex >= 0) {
             // V2
+            // TODO: 22/06/2017 Is any of this necessary?
```
Comment (riaanjvos) [inline app/src/main/java/piuk/blockchain/android/ui/account/AccountEditViewModel.java]: Wow. Not at all necessary

File: app/src/main/java/piuk/blockchain/android/data/datamanagers/PromptManager.kt (hunk 1 above)
Comment (ditn) [inline app/src/main/java/piuk/blockchain/android/data/datamanagers/PromptManager.kt]: Hahaha, soon you'll be forgetting them in Java instead

File: app/src/main/java/piuk/blockchain/android/ui/account/AccountEditViewModel.java (hunk 2 above)
Comment (ditn) [inline app/src/main/java/piuk/blockchain/android/ui/account/AccountEditViewModel.java]: Sweet, lets get rid of this stuff then.

**Pull request merged**